tests/components/todoist/test_calendar.py

```diff
@@ -42,3 +46,14 @@ def test_parse_due_date_without_timezone_uses_offset():
     }
     actual = _parse_due_date(data, timezone_offset=-8)
     assert datetime(2022, 2, 2, 22, 0, 0, tzinfo=dt.UTC) == actual
+
+
+def test_calendar_entity_unique_id():
+    """Test unique id is set to project id."""
+    entity = TodoistProjectEntity(
```

Comment:
Please set up the integration, while patching the library appropriately to avoid I/O, and get the entity entry from the entity registry and assert its state.

https://developers.home-assistant.io/docs/development_testing#writing-tests-for-integrations

Reply:
Thank you, I somehow had never seen those docs, likely because this component is mostly untested.